### ISSUE TITLE
It ain’t working in my browser..

### ISSUE THREAD

**Issue by abood1703** [faahim/happy-birthday#6]
I cloned the repo, opened chrome and navigated to the the file, chose index.html and it was blank. I tried commenting the css link line and it worked without css!! Please help!! I got her birthday on 27th 😬😓

**Comment by Creativty** [comment]
I am really just speculating, but I think that the problem may be **CORS** request, so I would recommend hosting or running it on a local server and trying again. Because just opening the file directly with chrome will most likely end up in **CORS** request errors.

**Comment by abood1703** [comment]
@Creativty thank you so much!! It worked in localhost!! 😅

**Comment by faahim** [comment]
Sorry, didn't notice it sooner! 😅 

@Creativty is correct, it doesn't link the javascript file correctly due to CORS when you open the HTML directly. But that's because I was loading the script through babel (for transpiling). Reading this issue, I reconsidered if I really need that anymore. It's almost 2021 and the code used in the project is supported by all modern browser out of the box now, I believe.
So I dropped it. 🙂 

Loading the HTML file directly will work now. The JSON file doesn't work that way though. You'll still need to run it on a (local)hots to get that working.

Hope you got to make it work in time @abood1703? 😃
